samples/bluetooth/direct_test_mode/src/fem/fem.h

```diff
@@ -154,6 +134,24 @@ uint32_t fem_default_active_delay_calculate(bool rx,
  */
 int fem_antenna_select(enum fem_antenna ant);
 
+/**@brief Function for getting a radio ramp-up time in the transmit mode.
```

Comment:
```suggestion
/**@brief Get radio ramp-up time in the transmit mode.
```
See https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#functions

Reply:
Done